packages/valory/agents/trader/aea-config.yaml

```diff
@@ -192,6 +193,16 @@ models:
         "openai-gpt-3.5-turbo", "openai-gpt-4", "stabilityai-stable-diffusion-v1-5",
         "stabilityai-stable-diffusion-xl-beta-v2-2-2", "stabilityai-stable-diffusion-512-v2-1",
         "stabilityai-stable-diffusion-768-v2-1"]}
+      use_nevermined: ${bool:true}
+      mech_to_subscription_params: ${list:[["base_url", "https://marketplace-api.gnosis.nevermined.app/api/v1/metadata/assets/ddo"],
```

Comment:
Maybe `mech_subscription_params`?

Reply:
there are various fixes to be made to the subscription params, I suggest we handle them in a future version

Reply:
#208